clients/src/main/java/org/apache/kafka/common/serialization/Serdes.java

```diff
@@ -48,6 +59,11 @@
 
     static public final class DoubleSerde implements Serde<Double> {
         @Override
+        public void configure(Map<String, ?> configs, boolean isKey) {
+            // do nothing
```

Comment:
Thinking of it again, I think I can just keep the `final private ser/deser` objects for all these cases, and call their corresponding configure functions.